DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/platform_designer_standard/README.md

```diff
@@ -0,0 +1,401 @@
+
+![](assets/csr-output-example-simple.svg)
+
+This example is intended for users interested in creating standalone modules that can be included in Intel® Quartus® Prime projects. It serves as a minimal example, and while it targets a specific board, a user familiar with the Intel® Quartus® Prime Software Suite should be able to easily port this design to other hardware.
+
+## Key Implementation Details
+
+This tutorial is structured with four source code directories.
+
+| Source Directory  | Description
+|:--                |:--
+| `add_oneapi`      | The source and build scripts needed to compile a simple IP using oneAPI
+| `add_quartus_sln` | An example of the project files that are created using the Intel® Quartus® Prime GUI
+| `starting_files`  | The minimal source files you need to create a Intel® Quartus® Prime project. Additional project files are created using the Intel® Quartus® Prime GUI.
+| `system_console`  | Scripts for controlling the System Console application while you test the design in hardware
+
+### Board-specific Considerations
+
+This design is intended to work with the Intel® Cyclone® V SoC FPGA. These board-specific configurations are not guaranteed to work with different boards:
+1. Choose `5CSEMA5F31C6` device to match the devkit
+2. Choose pin `PIN_AF14` to drive the `i_clk` signal
+3. Choose pin `PIN_V16` to drive the `fpga_led` signal
+4. Choose pin `PIN_AA14` to drive the `reset_button_n` signal
+
+## Building the `platform_designer` Tutorial
+
+> **Note**: When working with the command-line interface (CLI), you should configure the oneAPI toolkits using environment variables. 
+> Set up your CLI environment by sourcing the `setvars` script located in the root of your oneAPI installation every time you open a new terminal window. 
+> This practice ensures that your compiler, libraries, and tools are ready for development.
+>
+> Linux*:
+> - For system-wide installations: `. /opt/intel/oneapi/setvars.sh`
+> - For private installations: ` . ~/intel/oneapi/setvars.sh`
+> - For non-POSIX shells, like csh, use the following command: `bash -c 'source <install-dir>/setvars.sh ; exec csh'`
+>
+> Windows*:
+> - `C:\"Program Files (x86)"\Intel\oneAPI\setvars.bat`
+> - Windows PowerShell*, use the following command: `cmd.exe "/K" '"C:\Program Files (x86)\Intel\oneAPI\setvars.bat" && powershell'`
+>
+> For more information on configuring environment variables, see [Use the setvars Script with Linux* or macOS*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-linux-or-macos.html) or [Use the setvars Script with Windows*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-windows.html).
+
+**Follow these steps to compile and test the design:**
```

Comment:
```suggestion
**Follow these steps to compile and test the design:**
```
why did you bold this text? if you bold this, you should also bold the Quartus Pro version so they are consistent